post-office-stub/samconfig.toml

```diff
@@ -0,0 +1,11 @@
+version = 0.1
+[dev]
+[dev.deploy]
+[dev.deploy.parameters]
+stack_name = "f2f-post-office-stub"
+s3_prefix = "f2f-post-office-stub"
+region = "eu-west-2"
+confirm_changeset = false
+capabilities = "CAPABILITY_IAM"
+parameter_overrides = "CodeSigningConfigArn=\"none\" Environment=\"dev\" PermissionsBoundary=\"none\" SecretPrefix=\"none\" VpcStackName=\"vpc-cri\" L2KMSStackName=\"infra-l2-kms\""
```

Comment:
I don't think we need L2KMSStackName? Not seeing it being used in the template file

Reply:
Fixed